CHANGES.rst

```diff
@@ -89,6 +89,11 @@ New Features
     appropriate logarithmic quantity class if ``subok=True``. For instance,
     ``Quantity(1, u.dex(u.m), subok=True)`` yields ``<Dex 1.0 dex(m)>``. [#5928]
 
+  - The ``quantity_input`` decorator now accepts a string physical type instead
```

Comment:
this sounds a bit like a replacement rather than added functionality; maybe replace "instead of" with "in addition to"